core/src/components/item/test/basic/item.e2e.ts

```diff
@@ -10,5 +10,37 @@ configs().forEach(({ title, screenshot, config }) => {
 
       await expect(page).toHaveScreenshot(screenshot(`item-diff`));
     });
+
+    /**
+     * This behavior needs to be tested for all modes & directions
+     * Safe padding should stay on the same side when the direction changes
+     */
+    test('should have safe area padding', async ({ page }) => {
+      await page.setContent(
+        `
+        <style>
+          :root {
+            --ion-safe-area-left: 40px;
+            --ion-safe-area-right: 10px;
```

Comment:
```suggestion
            --ion-safe-area-left: 40px;
            --ion-safe-area-right: 20px;
```

Nit pick but we have used these values in a few other places, could we update them to match? 

https://github.com/ionic-team/ionic-framework/blob/b7d1a5c86bf81b0808c343f35ba64214dba39e6f/core/src/components/list-header/test/basic/list-header.e2e.ts#L29-L30
https://github.com/ionic-team/ionic-framework/blob/b7d1a5c86bf81b0808c343f35ba64214dba39e6f/core/src/components/fab/test/safe-area/fab.e2e.ts#L11-L12